### PR TITLE
Users/t anmah/pylint import error fix

### PR DESCRIPTION
**Pull request by andreamah**
# Description:
Update: I took another look at the pylint import error and I think I was able to get init-hook to recognize the libraries that are local to the extention!! Originally, I couldn't figure it out and closed the original PR.

Original Issue: All references to the local libraries (like adafruit_circuitplayground) were flagged as missing imports by pylint.

## Type of change

- [ x ] Bug fix (non-breaking change which fixes an issue)

# Limitations:

# Testing:

- [ ] Test with importing local libraries + checks with non-installed libraries to see if import errors still come up

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My code has been formatted with `npm run format` and passes the checks in `npm run check`
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -973,23 +977,51 @@ const checkForTelemetry = (sensorState: any) => {
...
+                        `import sys; sys.path.extend([\"${outPath}\",\"${pyLibsPath}\"])`
+                    ], 
+                    vscode.ConfigurationTarget.Workspace)
+}
+
+const createEscapedPath = (pieces: string[]) => {
+  const initialPath:string = pieces.join("\\");
+
+  // escape all instances of backslashes
+  return initialPath.replace(/\\/g,"\\\\");
```
Comment (isadorasophia) [inline src/extension.ts]: You can also use:
 
```python
import re
re.escape(str)
```

Assuming we want to escape for regex?

**Review by isadorasophia — APPROVED** [review]
yay :D

File: src/extension.ts (hunk 1 above)
Comment (andreamah) [reply src/extension.ts]: I tried to use escape(), but it also escaped other characters (like semicolons) :(

Comment (andreamah) [reply src/extension.ts]: ^ oh wait, your response is Python (I think). I'm using Typescript.

File: src/extension.ts [hunk 2]
```diff
@@ -973,23 +977,51 @@ const checkForTelemetry = (sensorState: any) => {
...
-        currentExtraPaths.push(pathToLib);
-    }
-    vscode.workspace
-        .getConfiguration()
-        .update(
-            "python.autoComplete.extraPaths",
-            currentExtraPaths,
-            vscode.ConfigurationTarget.Global
-        );
+  updateConfigLists("python.autoComplete.extraPaths",[__dirname], vscode.ConfigurationTarget.Global)
```
Comment (vandyliu) [inline src/extension.ts]: run prettier formatting 🥺

**Review by vandyliu — CHANGES_REQUESTED** [review]
run prettier & should be good 💃

File: src/extension.ts (hunk 2 above)
Comment (andreamah) [reply src/extension.ts]: done!

File: src/extension.ts (hunk 1 above)
Comment (isadorasophia) [reply src/extension.ts]: We talked offline, we don't really need to escape this for regex, but rather just the backlashes.

**Review by isadorasophia — APPROVED** [review]

**Review by vandyliu — APPROVED** [review]

**Pull request merged**